### PR TITLE
Update m365-dr-workload-viva-topics.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the migration wordings 

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10666

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

File: microsoft-365/enterprise/m365-dr-workload-viva-topics.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Please refer to the [ADR Commitment page](m365-dr-commitments.md#viva-topics) fo
 
 ## Migration
 
-Data is stored is maintained within Exchange Online, SharePoint Online and Microsoft Teams.  Migration processes are handled by the applicable/relevant workloads.
+Data stored is maintained within Exchange Online, SharePoint Online and Microsoft Teams.  Migration processes are handled by the applicable/relevant workloads.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/m365-dr-workload-viva-topics.md]:
```suggestion
Data stored is maintained within Exchange Online, SharePoint Online, and Microsoft Teams. Migration processes are handled by the applicable/relevant workloads.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@msbemba please pass the checks. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**